tooling/charts/tl500-course-content/values.yaml

```diff
@@ -37,6 +37,9 @@ crw:
   namespace: tl500-workspaces
   name: "codeready-workspaces"
 #  devRegImage: "quay.io/rht-labs/devfileregistry:test"
+#  selfSignedCert: false
+#  gitSelfSignedCert: false
+#  ServerTrustStoreConfigMapName: custom-ca-cert
```

Comment:
probably want this to be `che-git-self-signed-cert` to reflect the above hardcoded value, or otherwise use this variable for the name field in the CM